source/speech/__init__.py

```diff
@@ -1071,14 +1064,17 @@ def speakTextInfo(
 	)
 
 	if reason == controlTypes.REASON_SAYALL:
-		# Deprecation warning: In 2021.1 speakTextInfo will no longer  send speech through speakWithoutPauses
-		# if reason is sayAll, as sayAllhandler does this manually now.
-		return _speakWithoutPauses.speakWithoutPauses(speechSequences)

```

Comment:
This was a bug, `speechSequences` is a generator (now renamed to `speechGen`) but `speakWithoutPauses` does not accept a generator. It needs to be able to get the number of items in the sequence and slice it.